flocker/control/test/test_protocol.py

```diff
@@ -163,9 +175,12 @@ def test_connection_made_send_cluster_status(self):
         When a connection is made the cluster status is sent to the new client.
         """
         sent = []
-        self.patch(self.protocol, "callRemote",
-                   lambda *args, **kwargs: sent.append((args, kwargs))
-                   or succeed(None))
+        self.patch(
```

Comment:
This and latter variants are all the same, just with different lists. Maybe the utility method should be something that does patching? `self.patch_callremote(sent)`.